.github/workflows/scripts/monitor-typescript.ts

```diff
@@ -111,6 +119,7 @@ const transaction = Sentry.startTransaction({
 
 transaction.setName('typescript.compile');
 
+// @ts-ignore setMeasurements does not exist?
```

Comment:
probably the type is missing

Reply:
The type is a Transaction type, so it should work